.travis.yml

```diff
@@ -141,7 +141,7 @@ install:
           export SYMFONY_DEPRECATIONS_HELPER=weak &&
           cp composer.json composer.json.orig &&
           echo -e '{\n"require":{'"$(grep phpunit-bridge composer.json)"'"php":"*"},"minimum-stability":"dev"}' > composer.json &&
-          php .github/build-packages.php HEAD^ $COMPONENTS &&
+          (php .github/build-packages.php HEAD^ $COMPONENTS || true) &&
```

Comment:
why ? If this fails, the build will be garbage

Reply:
Because it does trigger a fatal on git and because of that trigger a fatal on php

Reply:
If we have a better way for this, maybe by fixing github/build-packager.php:11
```
$mergeBase = trim(shell_exec(sprintf('git merge-base %s HEAD', array_shift($dirs))));
```
that triggers a git fatal then a seg fault than a `exit 139` which lead to a build failure

Reply:
I'm trying to have the test running and we should be discussing this

Reply:
```
/home/travis/.travis/job_stages: line 57: 14926 Segmentation fault      (core dumped) $COMPOSER_UP
The command "if [[ ! $skip ]]; then $COMPOSER_UP; fi" failed and exited with 139 during .
```